build.gradle

```diff
@@ -243,7 +243,6 @@ task runGitPreCommitTasks {
   dependsOn 'test'
   dependsOn 'pmdMain'
   dependsOn 'pmdTest'
-  dependsOn 'dependencyCheckAggregate'
```

Comment:
Should this be committed?

Reply:
pipeline runs the dependency check, so removing it.